README.md

```diff
@@ -1,21 +1,139 @@
+
+## Install
+
+In order to install the `ipfscluster-server` the `ipfscluster` tool  simply download this repository and run `make` as follows:
+
+```
+$ go get -u -d github.com/ipfs/ipfs-cluster
+$ cd $GOPATH/src/github.com/ipfs/ipfs-cluster
+$ make install
+```
+
+This will install `ipfscluster-server` and `ipfscluster` in your `$GOPATH/bin` folder.
+
+## Usage
+
+### `ipfscluster-server`
+
+`ipfscluster-server` runs a member node for the cluster. Usage information can be obtained running:
+
+```
+$ ipfscluster-server -h
+
+```
+
+Before running `ipfscluster-server` for the first time, initialize a configuration file with:
+
+```
+$ ipfscluster-server -init
+```
+
+The configuration will be placed in `~/.ipfs-cluster/server.json` by default.
+
+You can add the multiaddresses for the other members of the cluster in the `cluster_peers` variable. For example, here is a valid configuration for a cluster of 4 members:
+
+```json
+{
+    "id": "QmSGCzHkz8gC9fNndMtaCZdf9RFtwtbTEEsGo4zkVfcykD",
+    "private_key" : "<redacted>",
+    "cluster_peers" : [
+          "/ip4/192.168.1.2/tcp/9096/ipfs/QmcQ5XvrSQ4DouNkQyQtEoLczbMr6D9bSenGy6WQUCQUBt",
+          "/ip4/192.168.1.3/tcp/9096/ipfs/QmdFBMf9HMDH3eCWrc1U11YCPenC3Uvy9mZQ2BedTyKTDf",
+          "/ip4/192.168.1.4/tcp/9096/ipfs/QmYY1ggjoew5eFrvkenTR3F4uWqtkBkmgfJk8g9Qqcwy51",
+          "/ip4/192.168.1.5/tcp/9096/ipfs/QmSGCzHkz8gC9fNndMtaCZdf9RFtwtbTEEsGo4zkVfcykD"
+        ],
+    "cluster_addr": "0.0.0.0",
+    "cluster_port": 9096,
+    "consensus_data_folder": "/home/user/.ipfs-cluster/data",
+    "api_addr": "0.0.0.0",
+    "api_port": 9094,
+    "ipfs_api_addr": "0.0.0.0",
+    "ipfs_api_port": 9095,
+    "ipfs_addr": "127.0.0.1",
+    "ipfs_port": 5001
+}
+```
+
+The configuration file should probably be identical among all cluster members, except for the `id` and `private_key` fields. To facilitate configuration, `cluster_peers` includes its own address, but it does not have to.
+
+Once every cluster member has the configuration in place, you can run `ipfscluster-server` to start the cluster.
+
+
+### `ipfscluster`
+
+`ipfscluster` is the client application to manage the cluster servers and perform actions. `ipfscluster` uses the HTTP API provided by the server nodes.
+
+After installing, you can run `ipfscluster --help` to display general description and options, or alternatively `ipfscluster help [cmd]` to display
+information about supported commands.
+
+In summary, it works as follows:
+
+```
+$ ipfscluster member ls                                                # list cluster members
+$ ipfscluster pin add Qma4Lid2T1F68E3Xa3CpE6vVJDLwxXLD8RfiB9g1Tmqp58   # pins a Cid in the cluster
+$ ipfscluster pin add Qma4Lid2T1F68E3Xa3CpE6vVJDLwxXLD8RfiB9g1Tmqp58   # unpins a Cid from the cluster
```

Comment:
There still is an issue here, the command `ipfscluster pin add` is repeated twice, for both pinning an unpinning. I believe this line should mention `ipfscluster pin rm` or similar.